packages/block-library/src/quote/style.scss

```diff
@@ -1,19 +1,19 @@
 .wp-block-quote {
+
 	&.is-style-large,
 	&.is-large {
-		margin: 0 0 16px;
```

Comment:
So, this is interesting to me, because currently in Gutenberg Quote block's `theme.scss` we're assigning a `margin-bottom` of `20px` on the overall `.wp-block-quote` wrapper. However, when the Quote block (again, current Gutenberg treatment) gets assigned as `.is-large` by the user, than we're _decreasing_ the `margin-bottom` from `20px` to `16px` here: https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/quote/style.scss#L2

This screams inconsistency in oversight to me. 😞 

Therefore, this is why I went to the extent of trying to update the overall styling of the Quote block component to a little more consistent and scaleable solution. In hopes that it would get merged before Gutenberg launch (WP 5.0).

Also, to clarify I changed `px` units to `em`, because using `px` is making things far too static, and has potential a11y concerns for users on both the back end and the front end.

Reply:
I deeply appreciate your work here, and while you have several valid points, as we barrel towards the November 27th release, it's critical that we focus pull requests only on solving very specific issues, and that we avoid lumping in unrelated changes. 

Even though there can be an argument for em changes, it's unrelated to the initial issue, and is only making this review more complex.

> So, this is interesting to me, because currently in Gutenberg Quote block's theme.scss we're assigning a margin-bottom of 20px on the overall .wp-block-quote wrapper (see line #2). However, when the Quote block (again, current Gutenberg treatment) gets assigned as .is-large by the user, than we're decreasing the margin-bottom from 20px to 16px here: 

Solid observation, worth fixing.